common/src/main/java/org/jboss/pnc/common/json/JsonUtils.java

```diff
@@ -32,4 +32,17 @@ public static String toJson(Object objectToBeMapped) throws JsonProcessingExcept
         return new ObjectMapper().writer().withDefaultPrettyPrinter().writeValueAsString(objectToBeMapped);
     }
 
+    /**
+     * @param key The name of the JSON field.
+     * @param value The value of the JSON field.
+     * @return If the value of the JSON field is null, the method returns: "<key>: null".
+     *         </p>
+     *         Otherwise it returns: "<key>": "<value>".
```

Comment:
inconsistent. one is wholy in quotes (that are not in the string), the other are not.

Reply:
:eagle: